sky/clouds/kubernetes.py

```diff
@@ -328,30 +367,46 @@ def _make(instance_list):
 
         # Currently, handle a filter on accelerators only.
         accelerators = resources.accelerators
+        default_instance_type = Kubernetes.get_default_instance_type(
```

Comment:
Should we increase the `DEFAULT_MEMORY_CPU_RATIO` when the GPU is involved? Users using SkyPilot to launch GPU tasks are likely running training or serving workloads that may require larger memory size than the same size as the number of cores of CPUs. Can we set it to 4 instead? Similar to the other clouds: https://github.com/skypilot-org/skypilot/blob/d00685e6c978435a656386fa05ff5bf0af1bcfb4/sky/clouds/service_catalog/gcp_catalog.py#L60